source/Calamari/Deployment/PackageRetention/Repositories/JsonJournalRepository.cs

```diff
@@ -84,5 +104,20 @@ public void Dispose()
         {
             semaphore.Dispose();
         }
+
+        static bool TryParseJournal(string json, out List<JournalEntry> result)
```

Comment:
I think this would be more accurately named if we called it `TryDeserializeJournalEntries` (just based off the JsonConvert method call)
You'll also need to change log message too.